### DISCUSSION TITLE
Amiga Assembly version 1.3.0 (pre-release) - 1.4.0 (stable)

### DISCUSSION THREAD

**Discussion by prb28** [prb28/vscode-amiga-assembly#203] [General]
New feature memory edition:
[memory edit](https://github.com/prb28/vscode-amiga-assembly/wiki/images/memoryedit.gif)
![](https://raw.githubusercontent.com/wiki/prb28/vscode-amiga-assembly/images/memoryedit.gif)

**Comment by Bippym** [comment]
This looks useful... One question, how do you colse it and return to normal debugging?

**Reply by prb28** [reply]
You are still in debugging.  Just close the window.

**Comment by Bippym** [comment]
WIth the new memory editor/viewer, if I click to view a variable shouldn't the memory viewer show me the memory where the variable is stored? It always starts at $0

See screenshot where I want to view the memory around struct: at $0x000264ba but it is showing me from $0

![Untitled](https://user-images.githubusercontent.com/21372489/154574647-ab3b92df-f19b-46f4-b4dd-430970c9210e.png)

**Reply by prb28** [reply]
You have to click on the little icon on the right side of the variable name.

**Reply by prb28** [reply]
Sorry, I've answered too quickly.
This options is a vscode standard. You must read the $0 as an offset from the chosen address in you case: 156858 in decimal.

**Comment by phaze101** [comment]
@prb28 is V1.4 available and what are the new enhancements exactly.

**Reply by prb28** [reply]
Not a lot for now, mostly the use of standard components of vscode:
- A memory visualiser/editor
- the default disassembled view 

I'm working on integrating vAmiga emulator.

**Reply by phaze101** [reply]
ok thanks. 

Can you look at Debugging when you have a minute. I was going to log an issue but I saw others has done this before me.

**Reply by prb28** [reply]
Which issue ?

**Reply by phaze101** [reply]
Variables are not always visible or you enter the name to watch it and it doesn't recognise it. 

I am using 1.3.6

BTW not sure but I think the title up here needs correction
Amiga Assembly version 1.3.0 is the stable

**Reply by prb28** [reply]
Do you have an example ? 
If you speak about this issue: https://github.com/prb28/vscode-amiga-assembly/issues/206 . It is a normal behavior, it is due to the relocation.

**Reply by phaze101** [reply]
@prb28 very similar but I did not use ORG for sure. I tried to replicate it yesterday and I couldn't lol. Ignore if it happens I will capture it again

**Comment by prb28** [comment]
# version 1.4.0 is release
- Fix binaries download issues
- Substitute variables in config args by *grahambates*
- Use of default vscode disassembly view non source debugging
- Add vscode hex editor for memory editing
- Bug fixes

For version 1.5.0 I'll try to release a first vAmiga integration.

**Reply by phaze101** [reply]
Good job
